docs/contributing/coding-guidelines/ros-nodes/console-logging.md

```diff
@@ -1,5 +1,183 @@
 # Console logging
 
-!!! warning
+ROS 2 logging is a powerful tool for understanding and debugging ROS nodes.
 
-    Under Construction
+This page focuses on how to design console logging in Autoware and shows several practical examples.
+To comprehensively understand how ROS 2 logging works, refer to the [logging documentation](https://docs.ros.org/en/humble/Concepts/About-Logging.html).
+
+## Logging use cases in Autoware
+
+- Developers debug code by seeing the console logs.
+- Vehicle operators take appropriate risk-avoiding actions depending on the console logs.
+- Log analysts analyze the console logs that are recorded in rosbag files.
+
+To efficiently support these use cases, clean and highly visible logs are required.
+For that, several rules are defined below.
+
+## Rules
+
+### Choose appropriate severity levels (required, non-automated)
+
+#### Rationale
+
+It's confusing if severity levels are inappropriate as follows:
+
+- Useless messages are marked as `FATAL`.
+- Very important error messages are marked as `INFO`.
+
+#### Example
+
+Use the following criteria as a reference:
+
+- **DEBUG:** Use this level to show debug information for developers. Note that logs with this level is hidden by default.
+- **INFO:** Use this level to notify events (cyclic notifications during initialization, state changes, service responses, etc.) to operators.
+- **WARN:** Use this level when a node can continue working correctly, but unintended behaviors might happen.
```

Comment:
perhaps with some examples in autoware:
(path optimization failed, low localization score, ego is surrond by obstacles, etc.)

Reply:
@alanmengg It's great, but let me confirm. Why is "path optimization failed" WARN?
Do you mean it failed but that it can use the previous data?

Reply:
How about this? 699f2483d11706e20dd8e2bea001c6a2cb1e08a8

Reply:
> @alanmengg It's great, but let me confirm. Why is "path optimization failed" WARN? Do you mean it failed but that it can use the previous data?

sometimes currently trajectory can't be optimized by obstacle avoidance module, but still can be executed with less smoothness.

Reply:
> How about this? [699f248](https://github.com/autowarefoundation/autoware-documentation/commit/699f2483d11706e20dd8e2bea001c6a2cb1e08a8)

It's great, seems more logical

Reply:
@alanmengg Can I merge this PR or do you have any other comments?

Reply:
> @alanmengg Can I merge this PR or do you have any other comments?

LGTM.
Thank u for the good work~👍